.github/debug.md

```diff
@@ -0,0 +1,47 @@
+# How to Perform Debugging
+
+The below instructions are written for Linux but include a method for using a remote debugger on Windows. Unless otherwise specified, the instructions are meant to be executed on a Linux machine.
+
+1. Clone the code to a directory of your choosing with your favorite git client or `git clone https://github.com/dell/dellemc-openmanage-ansible-modules.git`
+2. Download the code separately with `ansible-galaxy collection install dellemc.openmanage`. On Ubuntu this defaults to `/home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/` The problem with this is that this location is not in PYTHONPATH which will cause problems with the debugger.
+3. To resolve the pathing issues move the openmanage collection to align with the rest of Dell's code which is in PYTHONPATH with `sudo mv /home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/ /usr/local/lib/python3.X/dist-packages/ansible_collections/dellemc/`. The path may be different on your system, but it should be placed with your other packages. Alternatively, you can add the directory `/home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/` to PYTHONPATH with `export PYTHONPATH=$PYTHONPATH:/home/<YOUR_NAME>/.ansible/collections`. How to do this on Windows with a remote debugger is shown below.
```

Comment:
Would prefer to remove this `How to do this on Windows with a remote debugger is shown below.`. if needed you can explain this step in windows debugger section
```suggestion
3. To resolve the pathing issues move the openmanage collection to align with the rest of Dell's code which is in PYTHONPATH with `sudo mv /home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/ /usr/local/lib/python3.X/dist-packages/ansible_collections/dellemc/`. The path may be different on your system, but it should be placed with your other packages. Alternatively, you can add the directory `/home/<YOUR_NAME>/.ansible/collections/ansible_collections/dellemc/openmanage/` to PYTHONPATH with `export PYTHONPATH=$PYTHONPATH:/home/<YOUR_NAME>/.ansible/collections`. 
```